workflows/files/ZWEWRF03.xml

```diff
@@ -80,13 +80,16 @@ Keep this variable empty if you do not want to copy the STC task.</description>
     <variable name="LAUNCH_COMPONENT_GROUPS" scope="instance" visibility="public">
         <label>Launch Component Groups</label>
         <abstract>Comma separated list of components should start</abstract>
-        <description>Comma separated list of components should start</description>
+        <description>GATEWAY will start the API mediation layer that includes the API catalog, the API gateway, and the API discovery service.
```

Comment:
instead of ... <that> includes ... we should use ...<, which> includes ...
the same is true for the following lines

Reply:
Ok,  I hope I got it what you meant right...